docs/installation.md

```diff
@@ -85,15 +88,14 @@ Then solaar can be run as /usr/local/bin/solaar.
 [pip]: https://en.wikipedia.org/wiki/Pip_(package_manager)
 
 
-
-### Running Solaar at Startup
+## Running Solaar at Startup
 
 Solaar is run automatically at user login using a desktop file,
 which may have been installed at `/etc/xdg/autostart/solaar.desktop`.
 If you manually install Solaar you may need to modify this automatic starting of Solaar.
 
 
-### Using PyPI
+## Using PyPI
```

Comment:
Btw, what is the difference between just pip or PyPI?

Reply:
`pip install --user .` installs from the current directory and requires downloading from this repository.
`pip install --user solaar` installs from the PyPI repository without having to download.